app/models/invite.rb

```diff
@@ -157,11 +156,20 @@ def self.generate(invited_by, opts = nil)
     invite.reload
   end
 
-  def redeem(email: nil, username: nil, name: nil, password: nil, user_custom_fields: nil, ip_address: nil)
+  def redeem(email: nil, username: nil, name: nil, password: nil, user_custom_fields: nil, ip_address: nil, session: nil)
```

Comment:
I'm not happy that I have to pass the session which should really be a controller concern through the model. However, refactoring the whole invites redemption pipeline is not easy so we can leave with this for now?